CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
 
 ## Next
 
+### New and Improved
+
+* The observation tag was added to `storage_bucket.proto` and `storage_bucket_service_proto` for telemetry purposes. If you enable telemetry and observation events, Boundary will now collect data about session recording and storage buckets.
```

Comment:
My suggestion would be to combined storage bucket and session recording together into one sentence like below:
```suggestion
* The observation tag was added to Session Recording and Storage Bucket proto messages for telemetry purposes. If you enable telemetry and observation events, Boundary will now collect data about session recording and storage buckets.
```

Reply:
This sounds great and is easier for the user to understand. 👍